packages/bcer-data-portal/app/src/hooks/useLeaflet.tsx

```diff
@@ -0,0 +1,338 @@
+  const makeMarkerToolTip = (l: BusinessLocation): string => {
+    return `<div style="text-align:center;"><b>${safeGuardHtmlString(
+      l.business?.businessName ?? l.business?.legalName
+    )}</b><br/>${safeGuardHtmlString(l?.geoAddress ?? l.addressLine1)}</div>`;
+  };
+
+  /**
+   * Tool tip string is injected as html. This function makes sure to safe guard against user input.
+   * @param text
+   * @returns
+   */
+  const safeGuardHtmlString = (text: string) => {
+    return (
+      text?.replace(/[`~!@#$%^&*()_|+\=?;:'",.<>\{\}\[\]\\\/]/gi, '') ?? ''
+    );
+  };
+
+  /**
+   * Removes all the existing marker on map
+   */
+  const removeAllMarkers = () => {
+    if (markers && map) {
+      markers.forEach((mkr) => {
+        mkr.remove();
+        map.removeLayer(mkr);
+      });
+      setMarkers([]);
+    }
+  };
+
+  /**
+   * @param m map
+   * Adds Tile style to map
+   */
+  const addTileLayer = (m: L.Map) => {
+    L.tileLayer(
+      `https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=${
+        config.mapBoxAccessToken || ''
+      }`,
+      {
+        attribution:
+          config.mapBoxAttribution ||
+          'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+        maxZoom: 18,
+        id: config.mapBoxId || 'mapbox/streets-v11',
+        tileSize: 512,
+        zoomOffset: -1,
+        accessToken: config.mapBoxAccessToken || '',
+      }
+    ).addTo(m);
+  };
+
+  /**
+   * When user click on a location, map zooms on that location
+   * @param l
+   */
+  const showOnMapHandler = (l: BusinessLocation) => {
+    const { latitude, longitude } = l;
+    if (map && latitude && longitude) {
+      map.setView([+latitude, +longitude], 15);
+    }
+  };
+
+  /**
+   * resets Map if there is any change in map div dimension
+   */
+  const resetMapSize = () => {
+    if (map) {
+      map.invalidateSize();
+    }
+  };
+
+  /**
+   *
+   * @param message
+   * Displays error toast to the user
+   */
+  const showMapError = (message?: string) => {
+    setAppGlobalContext({
+      ...appGlobal,
+      networkErrorMessage: message || 'Map API did not load properly',
+    });
+  };
+
+  /**
+   * Get's the direction data based on locations and options selected
+   * @returns
+   */
+  const getDirectionData = async () => {
+    let count = 0;
+    if (startingLocation?.geometry) {
+      count++;
+    }
+    count += selectedLocations.length;
+
+    if (count < 2) {
+      return;
+    }
+
+    const url = new BcRouteLinkBuilder(config.bcDirectionApiKey)
+      .addPoints(startingLocation, selectedLocations)
+      .addOptions(routeOptions)
+      .build();
+
+    await fetchDirection(encodeURI(url));
+  };
+
+  /**
+   * To Fetch direction data from bc government service
+   * AxiosGet adds the Authorization header by default and
+   * that was causing the request to fail so using fetch instead
```

Comment:
I think you should be able to use `axios.get` instead of `AxiosGet` to make a request without the Auth header. Just to keep things consistent